src/ClientRuntime/global.json

```diff
@@ -1,5 +1,3 @@
 {
-  "projects": [ "Microsoft.Rest.ClientRuntime", "Microsoft.Rest.ClientRuntime.Azure.Authentication", "Microsoft.Rest.ClientRuntime.Etw", 
-                "Microsoft.Rest.ClientRuntime.Tests", "Microsoft.Rest.ClientRuntime.Azure", "Microsoft.Rest.ClientRuntime.Azure.Tests",
-                "Microsoft.Rest.ClientRuntime.Log4Net", "Microsoft.Rest.ClientRuntime.Tracing.Tests" ]
+  "projects": [ "../TestFramework" ]
```

Comment:
@hovsepm maybe I am not sure I understand your concern here, for test projects we will have to take dependency on TestFramework